src/main/resources/script_templates/ImageJ2-Scripts/Crop.py

```diff
@@ -0,0 +1,29 @@
+from net.imagej.axis import Axes
+from net.imglib2.util import Intervals
+ 
+from .ij_utils import get_axis
+ 
```

Comment:
I tried to run from Fiji script editor and get ' Attempted relative import in non-package'